src/GenFacades/GenFacades.Core/GenFacades.cs

```diff
@@ -489,6 +489,14 @@ public Assembly GenerateFacade(IAssembly contractAssembly, IAssemblyReference se
                             return forwardedTypes.TryGetValue(oldType.DocId(), out newType) ? newType : oldType;
                         });
 
+                        var remainingTypes = assembly.AllTypes.Where(t => t.Name.Value != "<Module>");
+
+                        if (!remainingTypes.Any())
+                        {
+                            Trace.TraceInformation($"Removed all types from {contractAssembly.Name} thus will remove ReferenceAssemblyAttribute.");
+                            assembly.AssemblyAttributes.RemoveAll(ca => ca.FullName() == "System.Runtime.CompilerServices.ReferenceAssemblyAttribute");
```

Comment:
IIRC, this is informational, and the real bit lives in assemblyflags. Am I misremembering or did the compiler/runtime change?

Reply:
correct the assemblyflags is the real bit.

Reply:
but we should also remove the ReferenceAssemblyAttribute.

Reply:
I see, let me strip that as well.

Reply:
We're defining that as 0x70, are all those bits part of the reference assembly flags?  I can't find this in the ecma spec.

Reply:
It uses both the attribute and the flags:
https://github.com/dotnet/coreclr/blob/13e7c4368da664a8b50228b1a5ef01a660fbb2dd/src/vm/pefile.inl#L128-L159

https://github.com/dotnet/coreclr/blob/cb8cfba3b61f18f81787322f0a2563d118c26b8a/src/inc/corhdr.h#L736-L779

Those flags actually correspond to the processor architecture.  0x70 means "no platform".  Clearing it means unspecified.